servers/src/main/java/tachyon/worker/block/evictor/EvictorType.java

```diff
@@ -30,6 +30,10 @@
    */
   LRU,
   /**
+   * Evict old blocks among several StorageDirs by PartialLRU
```

Comment:
Better to have a brief description here to tell the difference between LRU and PartialLRU.